public/_posts/_updates/yearn-allowlist/es.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Lista de permitidos de Yearn"
+image:
+  src: ./image1.jpg
+  width: 770
+  height: 367
+date: '2022-05-06'
+author: Weaver
+translator: Restlessmik3
+---
+
+![](./image1.jpg?w=770&h=367)
+
+## Visión general  
+
+Para hacer que el sitio web de Yearn sea aún más seguro, recientemente se ha introducido una nueva característica, denominada Lista de permitidos. En esencia, permite que todas las transacciones que se realizan a través del sitio web/SDK se validen con un contrato en cadena que mantiene un registro de todos los datos de llamadas válidos que el sitio web puede producir.
+
+Esto evita que el sitio web sea susceptible a varios ataques de intermediarios en los que se proporcionan datos al sitio web, ya sea a través de la API de Zapper/0x o de Yearn. Por ejemplo, si la API de Yearn devuelve una lista de vaults con direcciones maliciosas, con la intención de que los usuarios aprueben una de ellas cuando intentan depositar, la lista de permitidos bloquearía la realización de la transacción.
+
+Invitamos a otros protocolos a que también aprovechen sus propias listas de permitidos para mejorar su seguridad. Las instrucciones para crear su propia lista de permitidos se detallan más adelante.
+
+## Cómo funciona
+
+Tomemos un ejemplo de validación de los datos de llamada para una transacción que aprueba un depósito en una vault.
+
+En el contrato de la lista de permitidos hay una lista de condiciones, que son estructuras contra las que se validan.
+
+uí está la que se utiliza para aprobar un token en la vault (en formato JSON):
```

Comment:
AS-IS: uí está la que se utiliza para aprobar un token en la vault (en formato JSON):
TO-BE: aquí está la que se utiliza para aprobar un token en la vault (en formato JSON):

Reply:
Done